assets/js/base/components/button/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { Button as WPButton } from '@wordpress/components';
```

Comment:
I tested several packages to find out what was the best component to use here.

`@woocommerce/components` didn't have any `<Button>` component so it wasn't an option here.

The others packages with buttons were all very similar if you take out the style. I chose `@wordpress/components` because we will probably be using it in the checkout block (see #1351), so in some way it made sense to use the same library. `@automattic/components` and `@automattic/composite-checkout` also looked promising, but didn't seem to add anything that `@wordpress/components` didn't have, in addition `composite-checkout` is not in NPM yet and `@automattic/components` still has some issues when being used in external projects (see p90Yrv-1qf-p2 for more details).

However, if at some point we want to change, it will be pretty easy because the `<Button>` components of all those packages have a very similar API.

Reply:
Right, I noticed that as well, but I thought that was going to be solved in the next release thanks to @senadir's PR: https://github.com/WordPress/gutenberg/pull/18911?

Reply:
It will, but only if we import components directly from the package vs indirectly through the exported `wp.components` global (currently `@wordpress/components` is mapped to the global via the `DependencyExtractionWebpackPlugin`).  Hence my comment about handling things differently for frontend builds.

Reply:
Good point! I didn't think that they are externals so we can't force-update them to the last version.

Do you think we should open a follow-up issue to keep track of this and apply the solution commented in Slack?

Reply:
> Do you think we should open a follow-up issue to keep track of this and apply the solution commented in Slack?

Yup!